src/main/java/duke/Command.java

```diff
@@ -0,0 +1,130 @@
+package duke;
+import duke.exception.DukeException;
+import duke.tasks.*;
+
+import java.time.format.DateTimeParseException;
+
+
+public class Command {
+
+    private String[] cmd;
+
+    private boolean isExit = false;
```

Comment:
I feel that isExit is not descriptive enough. Is the command object an exit of what? Perhaps isExiting would be a better name for this.